src/main/java/appeng/container/implementations/ContainerInscriber.java

```diff
@@ -120,35 +120,32 @@ public boolean isValidForSlot( final Slot s, final ItemStack is )
 
 		if( s == this.middle )
 		{
-			for( final ItemStack optional : AEApi.instance().registries().inscriber().getOptionals() )
-			{
-				if( Platform.itemComparisons().isSameItem( optional, is ) )
-				{
-					return false;
-				}
-			}
-
 			boolean matches = false;
 			boolean found = false;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "found" closer to the code that uses it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1941)